tg_bot.py

```diff
@@ -52,6 +55,11 @@ def selecting_buro(update, context):
         'Hi! Here are available departments. Please select one:',
         reply_markup=InlineKeyboardMarkup(custom_keyboard, one_time_keyboard=True))
 
+    if user_id in scheduled_jobs:
```

Comment:
This check is repeated inside `print_subscription_status` as well. I suggest to do not check here (remove this condition), but keep the check inside the method

Reply:
Check removed in line 58, but kept on method, just `print_subscription_status` method is still there.